classes/options-general.php

```diff
@@ -720,37 +720,37 @@ class='howto'><?php _e( 'HTML5 - more precise, may prompt users for permission,
 
 									<div class="postbox">
 
-										<h3><?php _e( 'Map Style Defaults', 'simplemap' ); ?></h3>
+										<h3><?php _e( 'Map Style Defaults', 'SimpleMap' ); ?></h3>
 
 										<div class="inside">
-											<p class="sub"><?php printf( __( 'To insert SimpleMap into a post or page, type this shortcode in the body: %s', 'simplemap' ), '<code>[simplemap]</code>' ); ?></p>
+											<p class="sub"><?php printf( __( 'To insert SimpleMap into a post or page, type this shortcode in the body: %s', 'SimpleMap' ), '<code>[simplemap]</code>' ); ?></p>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 208 characters

<!-- Comment by [SideCI](https://sideci.com) -->